src/DomainServices/ProjectService.ts

```diff
@@ -430,4 +420,22 @@ export class ProjectService {
       }
     })
   }
+  private async processManuscriptModels(docs: Model[], containerID: string) {
+    const createdAt = Math.round(Date.now() / 1000)
+    return docs.map((doc) => {
+      const updatedDoc = {
+        ...doc,
+        createdAt,
+        updatedAt: createdAt,
+        containerID,
+      }
+
+      const errorMessage = validate(updatedDoc)
+      if (errorMessage) {
+        throw new SyncError(errorMessage, updatedDoc)
+      }
+
+      return updatedDoc
+    })
+  }
```

Comment:
The original method here in [ContainerService](https://github.com/Atypon-OpenSource/manuscripts-api/blob/4f4f885f39f265510378d90b81818ba671de4a7f/src/DomainServices/Container/ContainerService.ts#L729-L731)  updated the manuscriptID. Let me know if we need that, I did some debugging and it seems the models being sent already have the manuscriptID.
@mnatsheh, @hhusban

Reply:
Is this logic the same as the one in https://github.com/Atypon-OpenSource/manuscripts-api/blob/82997e54982c8b46b01ebf8a65003b56150c5939/src/DomainServices/ProjectService.ts#L127 ?

Reply:
> Technically yes, we might want to only update the updatedAt and validate the model in here, right?

That's fine. We should remove the validation from the `map` call and use `validate` with all the models (same as `importJats`).

We don't need to have this function `async` as far as I can tell.

Reply:
Done, I don't think QA testing is needed here, i'll merge directly to master if that's OK.